### DISCUSSION TITLE
next token feature

### DISCUSSION THREAD

**Discussion by shelu16** [projectdiscovery/uncover#150] [Q&A] [answered]
if  there is one token is give us error we can test to next token 

[shodan] unexpected status code 401 received from https://api.shodan.io/shodan/host/search?key=xxxxxx07e7-4e3a-8bf7-f0e7b7e783bf&query=jira&page=1

there is error in this token so check next token

**Comment by ehsandeep** [comment, marked as answer]
@shelu16 keys should be validated before putting into config file.
